.github/CODEOWNERS

```diff
@@ -1,22 +1,25 @@
 # Users referenced in this file will automatically be requested as reviewers for PRs that modify the given paths.
 # See https://help.github.com/articles/about-code-owners/
 
-/global.json                    @aspnet/build
-/.azure/                        @aspnet/build
-/.config/                       @aspnet/build
-/build/                         @aspnet/build
-/eng/                           @aspnet/build
+/global.json                    @dotnet/aspnet-build
+/.azure/                        @dotnet/aspnet-build
+/.config/                       @dotnet/aspnet-build
+/.github/                       @dotnet/aspnet-build
+/.github/*_TEMPLATE/            @dotnet/aspnet-build @mkArtakMSFT
+/build/                         @dotnet/aspnet-build
+/eng/                           @dotnet/aspnet-build
 /eng/common/                    @dotnet-maestro-bot
 /eng/Versions.props             @dotnet-maestro-bot @dougbu
 /eng/Version.Details.xml        @dotnet-maestro-bot @dougbu
-/src/Components/                @SteveSandersonMS
-/src/DefaultBuilder/            @tratcher @anurse
-/src/Hosting/                   @tratcher @anurse
-/src/Http/                      @tratcher @jkotalik @anurse
-/src/Middleware/                @tratcher @anurse
-/src/ProjectTemplates/          @ryanbrandenburg
```

Comment:
We don't have an assignment for this subfolder or lots of /src/ here or in 'main'. Do we need more ownerships❔

Reply:
Yeah, we should address that. @pranavkm for this one?